content/e20co/drone-marathon/index.md

```diff
@@ -78,7 +78,19 @@ Will result in:
 
 # Secrets
 
-Not Implemented.
+The Marathon plugin supports reading credentials from the Drone secret store. This is highly recommended instead of storing credentials in the pipeline configuration in plain text.
+
+```diff
+pipeline:
+  marathon:
+    image: e20co/drone-marathon:0.5
+    server: http://marathon.mesos:8080
+    values:
+-     DATABASE_PASSWORD: SomeSecretPassword
++     DATABASE_PASSWORD: ${DATABASE_PASSWORD}
```

Comment:
Oh dear... is there a "safer" way to format that string (keeping in mind that these values can be set to anything in the Secret store and then we'll pull these from the environment)?